lib/one_signal/subscription.ex

```diff
@@ -128,4 +118,11 @@ defmodule OneSignal.Subscription do
     |> put_params(params)
     |> make_request()
   end
+
+  def delete(subsctiption_id, opts \\ []) do
+    new_request(opts)
+    |> put_endpoint("/subscriptions" <> "/#{subsctiption_id}")
```

Comment:
Welp, I think I didn't hit "Start a review" when making my suggestion
```suggestion
  def delete(subscription_id, opts \\ []) do
    new_request(opts)
    |> put_endpoint("/subscriptions" <> "/#{subscription_id}")
```

Reply:
wow! nice catch!